site/app/site/scripts/views/form.js

```diff
@@ -29,6 +29,10 @@ define(["jquery", "underscore", "backbone", "models/account", "validation"],
                         window.location = "/site/maintenance";
                     }
                 });
+
+                // make first get request which fetches csrf token
+                Account.getUserSettings();
```

Comment:
AFAIU `initialize` is called each time html page is loaded, the idea is that if user is logged in 
`Account.getUserSettings()` will be intercepted by `CsrfPreventionRequestInteceptor` which will get the value from  _X-CSRF-Token_ header. If user is not logged in, it's not a big deal then request will fail unauthorized exception and token won't be fetched -  which is okay because unauthorized user is not vulnerable to CSRF attacks.
I considered another approach, which is to go for token when the first modification request is made but i found it more complex. 
Do you have any idea how to implement it in a better way?